docker/Dockerfile

```diff
@@ -1,27 +1,42 @@
 FROM python:3.9-slim
 
+# Keeps Python from generating .pyc files in the container
+ENV PYTHONDONTWRITEBYTECODE 1
+# Turns off buffering for easier container logging
+ENV PYTHONUNBUFFERED 1
+
 # Install essential tools
 RUN apt-get -y update && apt-get install -y \
     wget \
     gnupg \
     lsb-release
 
+# Install and setup poetry
+RUN pip install -U pip \
+    && apt install -y curl netcat \
```

Comment:
we really need `netcat` ? what are we using it for?

Reply:
Valid question. It seems to be that we are not using it.